components/molecule/thumbnail/src/index.scss

```diff
@@ -98,7 +103,7 @@ $ratios-molecule-thumbnail: (
 
     @each $key-ratio, $ratio in $ratios-molecule-thumbnail {
       &--#{$key-size}.sui-MoleculeThumbnail--#{$key-ratio}-#{$ratio} {
-        .sui-ImagePlaceholder {
+        .sui-AtomImage {
```

Comment:
Isn't this breaking AtomImage encapsulation?

Reply:
You're right. Removed `AtomImage` references in SASS